app/view/twig/files/_files.twig

```diff
@@ -12,7 +12,7 @@
 
     <tbody>
         {% for file in context.files %}
-            <tr>
+            <tr class="entry" data-bolt-browse-name="{{ file.filename|excerpt(80)|shy | trim }}">
```

Comment:
Did not see this earlier, sorry about that.
Currently the filename itself is trimmed down in the display, for BUIC as wel as the filemanager. 

It comes down to this, if the filename in the data-attribute would be the original (if longer than 80), than you'd get an inconsitancy in the display, the marking, and the search.

If you'd copy paste a filename in that is longer than 80 chars it would be found, but not marked.

If we'd remove the trim, we should also remove it for the display, so they are consistent. Simply left it there because I don't know why it was put in in the first place.